### DISCUSSION TITLE
Is issue 1230 a good feature to have?

### DISCUSSION THREAD

**Discussion by JonasMatos0** [kedacore/keda#1321] [Q&A / Need Help] [answered]
Hi, folks. 

I was looking for a Good First Issue to start collaborating on Keda and saw https://github.com/kedacore/keda/issues/1230

Supporting multi dimensions over cloudwatch scaler seems a nice feature to have, and apparently simple to implement. Is it a valid feature and also a Good First Issue to work on Keda? 

Thanks in advance.

**Comment by zroubalik** [comment, marked as answer]
Hi, absolutely, go ahead. And please comment on that issue, that you are taking it.
